kore/src/Kore/Step/Axiom/EvaluationStrategy.hs

```diff
@@ -299,17 +356,20 @@ evaluateWithDefinitionAxioms
     simplifier
     axiomIdToSimplifier
     patt
-  = do
+  =
+    AttemptedAxiom.exceptNotApplicable $ do
     let
+        -- TODO (thomas.tuegel): Take an 'ExpandedPattern' as input so that its
```

Comment:
Why an `ExpandedPattern` and not a `Predicate` or a `PredicateSubstitution`?

Reply:
That's a good point; I reworded the comment to leave that as an open question.